android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -256,41 +254,34 @@ public class ReactExoplayerView extends FrameLayout implements
     private long lastPos = -1;
     private long lastBufferDuration = -1;
     private long lastDuration = -1;
-
     private boolean viewHasDropped = false;
-    private void updateProgress() {
-        if (player != null) {
-            if (playerControlView != null && isPlayingAd() && controls) {
-                playerControlView.hide();
-            }
-            long bufferedDuration = player.getBufferedPercentage() * player.getDuration() / 100;
-            long duration = player.getDuration();
-            long pos = player.getCurrentPosition();
-            if (pos > duration) {
-                pos = duration;
-            }
 
-            if (lastPos != pos
-                    || lastBufferDuration != bufferedDuration
-                    || lastDuration != duration) {
-                lastPos = pos;
-                lastBufferDuration = bufferedDuration;
-                lastDuration = duration;
-                eventEmitter.progressChanged(pos, bufferedDuration, player.getDuration(), getPositionInFirstPeriodMsForCurrentWindow(pos));
-            }
-        }
-    }
-
-    private final Handler progressHandler = new Handler(Looper.getMainLooper()) {
-        @Override
-        public void handleMessage(Message msg) {
-            if (msg.what == SHOW_PROGRESS) {
-                updateProgress();
-                msg = obtainMessage(SHOW_PROGRESS);
-                sendMessageDelayed(msg, Math.round(mProgressUpdateInterval));
-            }
-        }
-    };
+//    private final Handler progressHandler = new Handler(Looper.getMainLooper()) {
```

Comment:
Why ? you are breaking progess bar with this change !
It is mandatory to revert this change !